misc/python/materialize/cloudtest/k8s/api/k8s_resource.py

```diff
@@ -34,7 +34,7 @@ def kubectl(self, *args: str, input: Optional[str] = None) -> str:
                 *args,
             ]
 
-            return subprocess.check_output(cmd, text=True, input=input)
+            return subprocess.run(cmd, text=True, input=input, check=True).stdout
```

Comment:
you may need to redirect stderr to stdout here. I think the `subprocess.run` documentation has an example on how to do that.

Reply:
Actually, `capture_output=False` by default, so I think everything will just be printed to stdout as it arrives, and nothing will be captured to be returned by `.stdout` ?

Reply:
In mzcompose/__init__.py you can see how `subprocess.run` is being called in case you want to capture the output and when you do not want to. My guess is that by default we do not want to capture anything, just let it end in stdout of the caller, unless explicitly specified/required.

Reply:
I changed it now to `stdout=subprocess.PIPE`, which is consistent with the behaviour of the deprecated function. This will both print the output and provide it as return value (as the previous version did).